RELEASE_PROCESS.md

```diff
@@ -244,3 +244,29 @@ branches were created on demand.
 This model has now largely been replaced by the model we see in
 this document. However you might still encounter leftovers form the old model
 and you are encouraged to create issues regarding inconsistencies.
+
+## Follow up actions
+
+Review and merge open pull requests.
```

Comment:
nit: This sounds like another subtitle (is this necessary)

Reply:
any suggestion?

```suggestion
Review and merge open pull requests at the following locations.
```